### PR TITLE
Added a simple onboarding screen.

### PR DESCRIPTION
**Pull request by adityathakurxd**
PR for Adding an Onboarding screen #6

I have used the `flutter_card_swipper: ^0.4.0` to create a swiping onboarding screen.
Also, set it as home in `main.dart` so it opens up first when user starts the app.

Let me know if any changes are required to the description.

### Video 

https://user-images.githubusercontent.com/53579386/123071211-16266380-d432-11eb-86bc-ff206ea0e788.mp4

### CODE REVIEW

File: .flutter-plugins [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # This is a generated file; do not edit or check into version control.
-camera=C:\\flutter\\.pub-cache\\hosted\\pub.dartlang.org\\camera-0.7.0+4\\
-wakelock=C:\\flutter\\.pub-cache\\hosted\\pub.dartlang.org\\wakelock-0.1.4+2\\
+camera=C:\\Users\\Asus\\AppData\\Roaming\\Pub\\Cache\\hosted\\pub.dartlang.org\\camera-0.7.0+4\\
+wakelock=C:\\Users\\Asus\\AppData\\Roaming\\Pub\\Cache\\hosted\\pub.dartlang.org\\wakelock-0.1.4+2\\
```
Comment (smaranjitghose) [inline .flutter-plugins]: @anushbhatia  I believe we can gitignore this file?

File: .flutter-plugins-dependencies [hunk 2]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+{"info":"This is a generated file; do not edit or check into version control.","plugins":{"ios":[{"name":"camera","path":"C:\\\\Users\\\\Asus\\\\AppData\\\\Roaming\\\\Pub\\\\Cache\\\\hosted\\\\pub.dartlang.org\\\\camera-0.7.0+4\\\\","dependencies":[]},{"name":"wakelock","path":"C:\\\\Users\\\\Asus\\\\AppData\\\\Roaming\\\\Pub\\\\Cache\\\\hosted\\\\pub.dartlang.org\\\\wakelock-0.1.4+2\\\\","dependencies":[]}],"android":[{"name":"camera","path":"C:\\\\Users\\\\Asus\\\\AppData\\\\Roaming\\\\Pub\\\\Cache\\\\hosted\\\\pub.dartlang.org\\\\camera-0.7.0+4\\\\","dependencies":[]},{"name":"wakelock","path":"C:\\\\Users\\\\Asus\\\\AppData\\\\Roaming\\\\Pub\\\\Cache\\\\hosted\\\\pub.dartlang.org\\\\wakelock-0.1.4+2\\\\","dependencies":[]}],"macos":[],"linux":[],"windows":[],"web":[]},"dependencyGraph":[{"name":"camera","dependencies":[]},{"name":"wakelock","dependencies":[]}],"date_created":"2021-06-23 14:39:50.782025","version":"2.3.0-17.0.pre.19"}
```
Comment (smaranjitghose) [inline .flutter-plugins-dependencies]: @anushbhatia  I believe we can gitignore this file?

**Review by smaranjitghose — APPROVED** [review]

**Review by anushbhatia — CHANGES_REQUESTED** [review]
@adityathakurxd yes add these .plugins .dependeicies in git ignore

**Comment by adityathakurxd** [comment]
@anushbhatia I have added these files to gitignore

**Comment by adityathakurxd** [comment]
I deleted the .flutter-plugins and .flutter-plugins-dependencies so they wouldn't merge on main.

**Review by smaranjitghose — APPROVED** [review]

**Review by anushbhatia — APPROVED** [review]

**Pull request merged**